apis/digitalocean_api.php

```diff
@@ -71,11 +71,11 @@ private function makePostConnection($action, array $post_data = NULL) {
         curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
         curl_setopt($ch, CURLOPT_HTTPHEADER, array(
             'Content-Type: application/json',
-            'Accept: application/json'
+            'Accept: application/json',
+			'Authorization: Bearer ' . $this->apiKey
```

Comment:
Can you fix the tabbing here? Each array element on its own line should be vertically aligned by the same indentation.